CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Products column change functionality - @vishal-7037 (#3017)
 - New Module order-history this provides the pagination via lazy laod - @hackbard (#2810)
 - OrderNumber on ThankYouPage - @Flyingmana (#2743)
+- Added possibility to change color or size of the product that is already in the cart - @andrzejewsky (#2346)
```

Comment:
this is PR to develop, so to newest version. You added it to 1.10 which was already released.
Please change it to 1.11 and read about [Releasing cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)